source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -682,6 +682,12 @@ void buildIntermediateResultForText(ACRView *rootView, ACOHostConfig *hostConfig
     auto markdownString = markDownParser->TransformToHtml();
     NSString *parsedString = (markDownParser->HasHtmlTags()) ? [NSString stringWithCString:markdownString.c_str() encoding:NSUTF8StringEncoding] : [NSString stringWithCString:markDownParser->GetRawText().c_str() encoding:NSUTF8StringEncoding];
 
+    if (markDownParser->HasHtmlTags() && ([parsedString containsString:@"\n"] || [parsedString containsString:@"\r"])) {
```

Comment:
Do we support complex syntax like this? If it's not handled carefully, possible you add wrong <br> into the output.
 

![image](https://user-images.githubusercontent.com/6290692/162284751-db8eead6-dab3-4e7c-8ca4-616dfd6690ff.png)

Reply:
that syntax is handled at the MarkDown parser in shared model.
His PR is handling line breaks after the text is parsed by the MarkDown parser.